qt-models/divepicturemodel.cpp

```diff
@@ -48,6 +51,29 @@ void DivePictureModel::updateDivePicturesWhenDone(QList<QFuture<void>> futures)
 	updateDivePictures();
 }
 
+void DivePictureModel::setZoomLevel(int level)
+{
+	zoomLevel = level / 10.0;
+	if (zoomLevel < -1.0)
+		zoomLevel = -1.0;
+	if (zoomLevel > 1.0)
+		zoomLevel = 1.0;
```

Comment:
zoom levels between -1 and +1  -- I think that deserves a comment that explains it?